docs/developers/guides/token.md

```diff
@@ -0,0 +1,198 @@
+# Launching a Token
+Welcome to our guide on launching a token collection using the Koinos Contract Standard (KCS-1) for Tokens and the AssemblyScript SDK for Koinos. In this tutorial, we'll walk you through the process of creating and deploying your own token on the Koinos blockchain. Whether you're a seasoned developer or new to blockchain development, this step-by-step guide will provide you with the knowledge and tools necessary to bring your token project to life. Let's dive in and explore the exciting world of token creation on Koinos!
+
+Before starting, ensure that you have already set up your Koinos AssemblyScript SDK environment by following [this guide](../as-sdk.md).
+
+---
+## Setting up the project
```

Comment:
Instead of cloning his examples, we should create the project from scratch similar to how it is done in the calculator example.